### PR TITLE
clarified market

### PR DESCRIPTION
**Pull request by Khrys-Samsonova**
Added more paragraphs

### CODE REVIEW

**Review by NickSheetz — DISMISSED** [review]

File: market.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-- Will want to cover most if not all needs
+In the future we plan on also extending this app across to the Otago Polytechnic and their locations. 
+  
   
-## Lecturers / Tutors 
+## Lecturers, Tutors and Other Staff
+With some anonymous data collection we could allow the university staff to see some statistics on where and when students are on campus. This could allow for some improvements to be made by the university. For example seeing how many people use the Library at 6am, and adjusting the opening time to later/earlier. 
+
 - Not a necessity but should try to make it work for these if we have the time
-- Maybe a way to track how student are going
```
Comment (Aaron-Rumpler) [inline market.md:14]: Maybe is the correct spelling.

File: market.md [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+
 
 ## Polytech Students
-- Will want to cover most if not all needs
+In the future we plan on also extending this app across to the Otago Polytechnic and their locations. 
+  
   
-## Lecturers / Tutors 
+## Lecturers, Tutors and Other Staff
+With some anonymous data collection we could allow the university staff to see some statistics on where and when students are on campus. This could allow for some improvements to be made by the university. For example seeing how many people use the Library at 6am, and adjusting the opening time to later/earlier.
```
Comment (Aaron-Rumpler) [inline market.md:16]: The university probably already has decent data on this from the WiFi.

**Review by bstabs-alt — DISMISSED** [review]
nice

File: market.md (hunk 1 above)
Comment (Aaron-Rumpler) [inline market.md:14]: ?

**Review by Aaron-Rumpler — CHANGES_REQUESTED** [review]

File: market.md [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+In the future we plan on also extending this app across to the Otago Polytechnic and their locations. 
+  
   
-## Lecturers / Tutors 
+## Lecturers, Tutors and Other Staff
+With some anonymous data collection we could allow the university staff to see some statistics on where and when students are on campus. This could allow for some improvements to be made by the university. For example seeing how many people use the Library at 6am, and adjusting the opening time to later/earlier. 
+
 - Not a necessity but should try to make it work for these if we have the time
-- Maybe a way to track how student are going
+- Maybe a way to track how students are going
```
Comment (Aaron-Rumpler) [inline market.md:19]: What do you mean by this? Welfare checks?

**Review by Aaron-Rumpler — CHANGES_REQUESTED** [review]

**Review by bstabs-alt — APPROVED** [review]
groovy

File: market.md (hunk 2 above)
Comment (NickSheetz) [reply market.md:16]: Whether we implement it or not isn't the point. Good to show we are thinking about all the possible implementations this app could have.

File: market.md (hunk 3 above)
Comment (Khrys-Samsonova) [reply market.md:19]: No. More so what I was talking about above with tracking usage of university locations.

Comment (Aaron-Rumpler) [reply market.md:19]: OK, so that might need to be clarified a bit.

Comment (Khrys-Samsonova) [reply market.md:19]: the whole point of the bullet points is to be a shorter version of what I stated above. so it already clarified...

File: market.md (hunk 2 above)
Comment (Aaron-Rumpler) [reply market.md:16]: Maybe something like 
> With some opt-in anonymous data collection we could potentially provide the University staff with additional statistics on where and when students are on campus.

File: market.md (hunk 3 above)
Comment (Aaron-Rumpler) [reply market.md:19]: It doesn't read that way to me. What does everyone else think?

File: market.md (hunk 2 above)
Comment (Khrys-Samsonova) [reply market.md:16]: yeah I'm happy with that if everyone else is

File: market.md (hunk 3 above)
Comment (Khrys-Samsonova) [reply market.md:19]: Could change it to "Maybe a way to track saturation of students at different locations around campus"

Comment (Aaron-Rumpler) [reply market.md:19]: That sounds much better.

**Review by Aaron-Rumpler — APPROVED** [review]

**Review by Aaron-Rumpler — APPROVED** [review]

**Review by bstabs-alt — APPROVED** [review]
good work

**Pull request merged**